graph/GRAPH.h

```diff
@@ -0,0 +1,123 @@
+#include<iostream>
+#include<vector>
+#include<queue>
+#include<list>
+#include<tuple>
+#include<set>
```

Comment:
The [contribution guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#typical-structure-of-a-program) requires that you briefly document your #includes with ///